NppMarkdownPanel/MarkdownPanelController.cs

```diff
@@ -178,7 +178,7 @@ private void ShowHelp()
         {
             StringBuilder sbPluginPath = new StringBuilder(Win32.MAX_PATH);
             Win32.SendMessage(PluginBase.nppData._nppHandle, (uint)NppMsg.NPPM_GETPLUGINHOMEPATH, Win32.MAX_PATH, sbPluginPath);
-            var helpFile = Path.Combine(sbPluginPath.ToString(), Main.PluginFilename + "\\README.md");
+            var helpFile = Path.Combine(sbPluginPath.ToString(), Main.PluginName + "\\README.md");
```

Comment:
* `Path.Combine` has [a 3-argument overload], so you can avoid the potentially fragile concatenation
* interpolating a variable implicitly calls `#ToString()`; not need to spell it out

~~~diff
- var helpFile = Path.Combine(sbPluginPath.ToString(), Main.PluginName + "\\README.md");
+ var helpFile = Path.Combine($"{sbPluginPath}", Main.PluginName, "README.md");
~~~


[a 3-argument overload]: https://learn.microsoft.com/dotnet/api/system.io.path.combine?view=netframework-4.5.2#system-io-path-combine(system-string-system-string-system-string)

Reply:
@rdipardo indeed it does!  You can tell my C# skills are very limited so I do appreciate your code review and input.  I tested and it works fine.  I pushed a new commit to this PR.